activerecord/lib/active_record/railties/databases.rake

```diff
@@ -2,8 +2,7 @@ require 'active_record'
 
 db_namespace = namespace :db do
   task :load_config do
-    ActiveRecord::Base.configurations       = ActiveRecord::Tasks::DatabaseTasks.database_configuration || {}
-    ActiveRecord::Migrator.migrations_paths = ActiveRecord::Tasks::DatabaseTasks.migrations_paths
```

Comment:
How will no longer setting this configuration affect other rake tasks? For example `db:rollback`. They all refer to `ActiveRecord::Migrator.migrations_paths`:
* https://github.com/rails/rails/blob/master/activerecord/lib/active_record/railties/databases.rake#L82
* https://github.com/rails/rails/blob/master/activerecord/lib/active_record/railties/databases.rake#L90
* https://github.com/rails/rails/blob/master/activerecord/lib/active_record/railties/databases.rake#L102
* ...